ch341a_spi.c

```diff
@@ -444,7 +444,11 @@ int ch341a_spi_init(void)
 		return -1;
 	}
 
-	libusb_set_debug(NULL, 3); // Enable information, warning and error messages (only).
+#if LIBUSB_API_VERSION >= 0x01000106
+   	libusb_set_option(NULL, LIBUSB_OPTION_LOG_LEVEL, LIBUSB_LOG_LEVEL_INFO);
+#else
+    libusb_set_debug(NULL, 3);
```

Comment:
@Asiderr Yout patch uses spaces not tabs for identation. Please take care of that.